service/src/main/java/uk/gov/hmcts/reform/fpl/events/InitiateCaseEvent.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.fpl.events;
+
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+
+public class InitiateCaseEvent {
```

Comment:
Because every single post submit event will be the same I would suggest creating base class and then having two subclasses, one for `InitiateCaseEvent` and other one for `SubmittedCaseEvent`.

```
CallbackEvent:
  - InitiateCaseEvent
  - SubmittedCaseEvent
```

We should also consider using the same wording format for all the events, at the moment one uses past tense other does not.